.github/workflows/build-system.yml

```diff
@@ -60,26 +60,20 @@ jobs:
           vulkan-components: Vulkan-Headers, Vulkan-Loader
           vulkan-use-cache: true
 
-      - name: Prepare build environment
-        run: |
-          pip3 install conan
-          conan config install https://github.com/NovelRT/ConanConfig.git
-
-      - name: Generate Build System
-        run: conan install . -if build --build=missing --profile ${{ matrix.profile }}-${{ matrix.configuration }}
-
       - uses: novelrt/setup-cpp@v1
-      - name: Build and Test NovelRT - ${{ matrix.configuration }}
+      - name: Build NovelRT - ${{ matrix.configuration }}
         id: build
         run: |
           ${{ matrix.c }} ${{ matrix.cxx }} cmake -S . -B build -DCMAKE_BUILD_TYPE=${{ matrix.configuration }}
-          cmake --build build --config ${{ matrix.configuration }}
-          cd build/tests/NovelRT.Tests
-          ctest --output-on-failure
+          cmake --build build --config ${{ matrix.configuration }} --target compiler Engine Interop Samples
 
-      - name: Install NovelRT - ${{ matrix.configuration }}
+      - name: Test NovelRT - ${{ matrix.configuration }}
         if: steps.build.outcome == 'success'
```

Comment:
What actually happened here? Can't make sense of it

Reply:
Removed the install step in CI and bumped it to only testing NovelRT after the engine, interop, samples, and Fabulist compiler are built